### DISCUSSION TITLE
Cannot create namespace

### DISCUSSION THREAD

**Discussion by kakarot-dev** [prisma/prisma#12670] [Q&A]
Does anyone know why this error is apprearing?
Im using the mongodb atlas
https://cdn.kakarot.pw/wvNIxkpzrK.png

**Comment by janpio** [comment]
Are you using MongoDB 4.2? Then unfortunately you can not create the collection via Prisma Client when creating data, but have to create it manually first.

**Reply by kakarot-dev** [reply]
oh ok, thx for the quick response!

**Reply by kakarot-dev** [reply]
well actually, the version is `5.0.6`...

**Reply by kakarot-dev** [reply]
The first time i tried was version `4.2`, then i switched to `5.0` which gave the same error until i manually created it...
Sometime later, it started showing the same error, but when this time i wiped the db, the error went away

**Reply by janpio** [reply]
That is weird. On 4.2+ this should work.

Can you reproduce the problem? And maybe create a reproduction repository?
Where are you hosting your MongoDB database?

**Reply by kakarot-dev** [reply]
at first, `clever-cloud` now i switched to atlas figuring originally this was a issue from clever-cloud

As for reproduction, ive been trying to reproduce it, but to no good, ill send you updates if i can reproduce it till tomorrow

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
